rog/rog/settings/base.py

```diff
@@ -259,6 +260,7 @@
 PAYMENT_IDS = os.getenv('PAYMENT_IDS', '123')
 PAYMENT_BASE_URL = os.getenv('PAYMENT_BASE_URL', 'https://testeplacila.si/vstop/index')
 REGISTRATION_NUMBER = os.getenv('REGISTRATION_NUMBER', '0000000')
+PANTHEON_URL = os.getenv('PANTHEON_URL', 'http://89.142.186.50:5000')
```

Comment:
tukaj daj neko neresnično vrednost, this is asking for trouble

Reply:
Kak je pa to pršlo notr :roll_eyes: ? Uredim